include/cpr/cookies.h

```diff
@@ -3,12 +3,38 @@
 
 #include "cpr/curlholder.h"
 #include <initializer_list>
-#include <map>
 #include <sstream>
 #include <string>
+#include <vector>
 
 namespace cpr {
 
+static const int EXPIRES_STRING_SIZE = 100;
```

Comment:
This is because `std::array` needs `const` value for it's declaration, and I will add the following comments:
```cpp
/**
 * EXPIRES_STRING_SIZE is an explicitly static and const variable that could be only accessed within the same namespace and is immutable.
 * To be used for "std::array", the expression must have a constant value, so EXPIRES_STRING_SIZE must be a const value.
 **/
static const std::size_t EXPIRES_STRING_SIZE = 100;
```